stacker/actions/base.py

```diff
@@ -79,8 +80,13 @@ def ensure_cfn_bucket(self):
                 logger.exception("Access denied for bucket %s.",
                                  self.bucket_name)
                 raise
+            elif e.response['Error']['Message'] == "Bad Request":
```

Comment:
Is this the correct case?  When another account is using a bucket name, I get the following:

```
>>> c.head_bucket(Bucket=USED_BUCKET)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/mike/venv/boto3/lib/python2.7/site-packages/botocore/client.py", line 251, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/Users/mike/venv/boto3/lib/python2.7/site-packages/botocore/client.py", line 537, in _make_api_call
    raise ClientError(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (403) when calling the HeadBucket operation: Forbidden
```